ix-dev/community/actual-budget/ix_values.yaml

```diff
@@ -0,0 +1,5 @@
+consts:
```

Comment:
We don't have image defined in ix_values for this ?

Reply:
Whoops, thanks for catching that!